operations/reservations.md

```diff
@@ -132,8 +133,9 @@ Returns all reservations within scope of the Access Token, filtered according to
 | `EndUtc` | string | required | Reservation end in UTC timezone in ISO 8601 format. |
 | `Number` | string | required | Confirmation number of the reservation in Mews. |
 | `State` | string [Service order state](#service-order-state) | required | State of the reservation. |
-| `Origin` | string [Service order origin](#service-order-origin) | required | Origin of the reservation. |
-| `OriginDetails`| string | optional | Details about the reservation [Origin](#service-order-origin). |
+| `Origin` | string [Service order origin](productserviceorders.md#service-order-origin) | required | Origin of the reservation. |
+| `CommanderOrigin` | string [Commander origin](#commander-origin) | optional | Specification of origin in case of Origin is `Commander`. |
```

Comment:
```suggestion
| `CommanderOrigin` | string [Commander origin](#commander-origin) | optional | Further detail about origin in case of Origin `Commander`. |
```